src/test/java/org/swisspush/reststorage/StorageExpandIntegrationTest.java

```diff
@@ -70,13 +71,14 @@ public void testPOSTWithWrongBody(TestContext context) {
                 .assertThat().statusCode(BAD_REQUEST)
                 .assertThat().body(equalTo(BAD_REQUEST_PARSE_MSG));
 
+        //Note: in Vertx 4 the wrong data type is acceptable
```

Comment:
propose to write here

Note: In Vertx 4 the integer is converted into a string and all 3 resources will be deleted

and actually put the resources there in order that we get a 200

Reply:
I changed, please see in diff, don't understand why not pickup at here

Reply:
Still propose to change the //Note: